lib/appraisal/bundler_dsl.rb

```diff
@@ -100,22 +100,28 @@ def ruby_version_entry
 
     alias_method :ruby_version_entry_for_dup, :ruby_version_entry
 
-    [:dependencies, :gemspec].each do |method_name|
-      class_eval <<-METHODS, __FILE__, __LINE__
-        private
+    def gemspec_entry
+      if @gemspecs.any?
+        @gemspecs.map(&:to_s).join("\n")
+      end
+    end
 
-        def #{method_name}_entry
-          if @#{method_name}
-            @#{method_name}.to_s
-          end
-        end
+    def gemspec_entry_for_dup
+      if @gemspecs.any?
```

Comment:
@jferris I don't know if you we notified of these changes because I just looked at this issue and Github asked me to "submit for review".